### PR TITLE
React hot loader error fix

### PR DESCRIPTION
**Pull request by jeremija**
An attempt to fix `react-hot-loader` `require()` error described in issue #47.

Would appreciate your feedback, @Macil!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -227,7 +227,13 @@ module.exports = function(bundle, opts) {
...
 
   function fileKey(filename) {
-    return path.relative(basedir, filename);
+    const relative = path.relative(basedir, filename);
+    if (/^[a-zA-Z0-9]/.test(relative)) {
+      // Example: return ./Label.jsx instead of Label.jsx - reverse module id
+      // lookup would not work with just Label.jsx
+      return './' + relative
+    }
+    return relative
```
Comment (Macil) [inline index.js]: Is this change still necessary with the initModule change? I can see how this change by itself could solve the issue in the specific case with multiple source files each in the basedir, but I think the initModule change alone should solve the general case.

Comment (jeremija) [reply index.js]: You're right, I do not think this change is necessary. I had thought it wasn't at first, but when I tested it, it didn't work. I've just tested it again with the original impl of `fileKey` and it seems to work (no error from `react-hot-loader`).

**Comment by Macil** [comment]
Thanks! I've just published this in version 0.4.1.

**Comment by jeremija** [comment]
Thank you!

**Pull request merged**